lib/feedparser/index.js

```diff
@@ -430,6 +431,7 @@ FeedParser.prototype.handleMeta = function handleMeta (node, type, options) {
 
   var meta = {}
     , normalize = !options || (options && options.normalize)
+    , parseDate = !options || (options && options.parseDate)
```

Comment:
`parseDate = !options || (options && options.parseDate)`

This logic will need to be flipped: `stringifydate` should be true only if `options && options.stringifydate` is true.